tests/test_html.py

```diff
@@ -397,6 +397,307 @@ def test_non_entity_unicode_character(self):
         self.assert_document_generates_html(document, expected_html)
 
 
+class FakedSubScriptTestCase(DocumentGeneratorTestCase):
+    style_xml = '''
+        <style styleId="foo" type="paragraph">
```

Comment:
Why you think `foo` couldn't show up? Do you mean something like `'Green'` or `'Base'` or `'TestStyle'` (these examples appear in the documentation).